### ISSUE TITLE
timeout=0 bugfix not on pypi

### ISSUE THREAD

**Issue by pettazz** [smontanaro/pylockfile#9]
It appears that the code for the current version on pypi is 0.9.1 and does not contain the bug fix for timeout=0. Can we get that updated?

**Comment by smontanaro** [comment]
I've asked a number of times in a couple different venues looking fire
someone to take over thus project. Would you like to?

**Comment by pettazz** [comment]
Sure why not?

**Comment by smontanaro** [comment]
Good. What do I need to do? Do you have a PyPI login? I'll work with those
folks to hand management of that record off to you as well.

Skip

**Comment by pettazz** [comment]
Not entirely sure. My pypi username is pettazz. Mind sending me an email so we can discuss?

**Comment by smontanaro** [comment]
> Not entirely sure. My pypi username is pettazz. Mind sending me an email so we can discuss?

Sure, though I don't know your email. You can mail me at
skip@pobox.com or skip@python.org to start up the dialog.

Skip

**Comment by gdub** [comment]
Note that the commit 58c3832d4b99cc33218c2d7af04a54272602b511 has the same bug as before, effectively doing this:

``` python
>>> timeout = 0
>>> timeout = timeout is not None and timeout or None
>>> timeout
>>> timeout is None
True
```

I've added a fix for this as part of #10

**Comment by roman-neuhauser** [comment]
@pettazz @smontanaro have you guys managed to move the maintainership thing forward?  downstream packagers like to have a stable upstream with blessed tarballs. :)

**Comment by smontanaro** [comment]
> @pettazz @smontanaro have you guys managed to move the maintainership thing forward? downstream packagers like to have a stable upstream with blessed tarballs. :)

Nope. Nobody has stepped up. I might have more time in the next month
to work on it.

**Comment by roman-neuhauser** [comment]
@smontanaro cool.  ITMT, do you still have old tarballs that used to be in googlecode somewhere?  any chance of restoring the old download urls?

curiously, the 0.9.1 tarball in pypi has a different checksum from the one that used to be in googlecode...

**Comment by coddingtonbear** [comment]
Hey there @smontanaro; if you need somebody to help out with this, I'm willing to lend a hand.

**Comment by ethanfurman** [comment]
pylockfile is no longer developed at this location.

It has been moved to:

https://git.openstack.org/cgit/openstack/pylockfile/

If the fix is not in that repo, I'm sure they wouldn't mind the help.  :)
